src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -120,6 +125,7 @@ def run_relevance_eval(
         be parsed.
     """
 
+    model._verbose = verbose
```

Comment:
I'm not sure I fully like the way a field is getting set this way because it's going to be inevitable that this flag gets left set - seems like it would be cleaner to parameterize as a kwarg on certain calls? That way there's no magic. There are going to be other code paths that will need to set this and doing it via parameters seems more scalable.

Reply:
ah yeah, that's a good point. If we want to be able to ask the model if it should emit verbose messages maybe we can hold the state in a context manager? Passing the arguments around is also a fine idea but I feel sometimes a flag that's passed around a lot of times can be really confusing to keep track of.